content/applications/finance/accounting/fiscal_localizations/localizations/colombia_ES.rst

```diff
@@ -42,27 +42,34 @@ Instalar a los primeros dos módulos:
    :align: center
 
 
-Configuración de las credenciales del Servicio Web de Carvajal T&S
+Configuración de las credenciales del Servicio Web de Carvajal
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Title specification should be shortened as well ;)

Reply:
Thanks, I totally missed that :sweat_smile: .